test/functional/wallet_send.py

```
@@ -35,7 +37,7 @@ def test_send(self, from_wallet, to_wallet=None, amount=None, data=None,
                   conf_target=None, estimate_mode=None, fee_rate=None, add_to_wallet=None, psbt=None,
                   inputs=None, add_inputs=None, include_unsafe=None, change_address=None, change_position=None,
                   include_watching=None, locktime=None, lock_unspents=None, subtract_fee_from_outputs=None,
-                  expect_error=None):
+                  expect_error=None, solving_data=None):
```

Ruff (0.12.2)

Missing type annotation for function argument `expect_error`

(ANN001)